src/ir/module.h

```diff
@@ -43,9 +43,9 @@ class Operation {
         inputs_(std::move(inputs)),
         impl_module_(std::move(impl_module)) {}
 
-  // Disable copy (and move) semantics.
+  // Disable copy semantics.
   Operation(const Operation&) = delete;
-  Operation& operator=(const Operation&) = delete;
+  Operation(Operation &&) = default;
```

Comment:
Why do you allow `operator=`?

Reply:
This was for another change, don't need this anymore. I've removed it.